lib/services/platform-service.ts

```diff
@@ -243,13 +244,19 @@ export class PlatformService implements IPlatformService {
 			emulatorServices.checkAvailability().wait();
 
 			this.buildPlatform(platform).wait();
-
 			var packageFile = this.getLatestApplicationPackageForEmulator(platformData).wait().packageName;
 			this.$logger.out("Using ", packageFile);
 
-			var logFilePath = path.join(platformData.projectRoot, this.$projectData.projectName, "emulator.log");
+			var projectName = options["debug-brk"] ? this.$projectData.projectName + "WithInspector" : this.$projectData.projectName;
+			var logFilePath = path.join(platformData.projectRoot, projectName, "emulator.log");
 
 			emulatorServices.startEmulator(packageFile, { stderrFilePath: logFilePath, stdoutFilePath: logFilePath }).wait();
+
+			if (platform === "ios" && options["debug-brk"]) {
```

Comment:
This `if` shouldn't be here. You should add new function in `IPlatformProjectService` interface - for example `runDebugger()` and implement it only in `ios-project-service`. You should be able to call it here - `platformData. platformProjectService.runDebugger()`.